src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+
+    public static int randome() {
+        Random myRandom = new Random();
+        return myRandom.nextInt(100);
+    }
+
+    public static Ball getRandomBall() {
```

Comment:
The method must not be static. Just create an object of Lotery class in main method